packages/runtime/container-runtime/src/summary/summarizerTypes.ts

```diff
@@ -295,7 +295,9 @@ export type SummarizerStopReason =
 	/** Summarizer client was disconnected */
 	| "summarizerClientDisconnected"
 	/* running summarizer threw an exception */
-	| "summarizerException";
+	| "summarizerException"
+	/* running summarizer threw an exception */
+	| "unexpectedSummaryAck";
```

Comment:
Maybe something like `latestSummaryStateStale`? The reason being this also happens when we explicitly try to download state from snapshot (2nd summary attempt) without an ack.

Reply:
or even latestSummaryStale